cylc/flow/cfgspec/globalcfg.py

```diff
@@ -409,6 +420,7 @@
             Examples::
 
                atom --wait
+               code --wait
                code -nw
```

Comment:
Duplicate entry, I think `-nw` (eqiv `--new-window --wait`) is better?

Reply:
IMO --long-form is almost always preferable, if you are writing it anywhere permanent.